### PR TITLE
Documentation v0.1.0

### PR DESCRIPTION
**Pull request by jenshaase**
## 📚 Description

Updated documentation for the first release

## 🔖 Changes

- Updated the getting started guide
- Improved Repl docs
- Add Changelog.md

### CODE REVIEW

**Review by Chemaclass — APPROVED** [review]
👌🏻

File: doc/content/_index.md [hunk 1]
```diff
@@ -54,4 +54,4 @@ Phel requires PHP 7.4 or higher and Composer. Read the [Getting Started Guide](/
 
 ## Status of Development
 
-Phel has not been released yet, but it is fairly complete. In the next weeks or months I will probably find some more edge cases that I will fix before the first official release. Maybe some of you are willing to test it out and give feedback.
+Phel has is fairly complete but not marked as ready. We still want to envolve this language without thinking to much about breaking changes. Maybe some of you are willing to test it out and give feedback.
```
Comment (JesusValera) [inline doc/content/_index.md:57]: `Phel has is`?
I think the `has` is not necessary.

File: doc/content/documentation/repl.md [hunk 2]
```diff
@@ -3,53 +3,64 @@ title = "REPL"
 weight = 18
 +++
 
-Phel provides a interactive prompt where you can type Phel expressions and see the results. This interactive prompt is called REPL (stands for Read-eval-print loop). A REPL is very helpful to test out small task or to play around with the language its self.
+Phel comes with an interactive prompt. The prompt accepts Phel expressions and directly returns the result. This interactive prompt is called REPL (stands for Read-eval-print loop). A REPL is very helpful to test out small task or to play around with the language its self.
```
Comment (JesusValera) [inline doc/content/documentation/repl.md:6]: Missing `s` in "to test out small taskS or..."?

`its self` doesn't exist, should be `itself`.

File: doc/content/documentation/repl.md [hunk 3]
```diff
@@ -3,53 +3,64 @@ title = "REPL"
...
+phel:1> (* 6 7)
 42
->>>
+phel:2>
+```
+
+The prompt also accepts multiline expressions:
+```bash
+Welcome to the Phel Repl
+Type "exit" or press Ctrl-D to exit.
```
Comment (JesusValera) [inline doc/content/documentation/repl.md:26]: This is out of context.
Why "Ctrl + D" instead of "Ctrl + C"?

File: doc/content/documentation/repl.md [hunk 4]
```diff
@@ -3,53 +3,64 @@ title = "REPL"
...
+10
+phel:4>
 ```
 
 Press `Ctrl-D` or type "exit" to end the REPL session.
 
 ## Little helpers
 
-The REPL itsself provides a few little helpers to make you life easier.
+The REPL itsself provides a few little helper functions.
```
Comment (JesusValera) [inline doc/content/documentation/repl.md:38]: Only one 's' in 'itself'. :)

**Review by JesusValera — APPROVED** [review]
Small typo suggestions.

File: doc/content/documentation/repl.md (hunk 3 above)
Comment (Chemaclass) [reply doc/content/documentation/repl.md:26]: Because it's what you see when entering the REPL 😄
Screenshot 2021-01-30 at 23 33 13

File: doc/content/documentation/repl.md [hunk 5]
```diff
@@ -3,53 +3,64 @@ title = "REPL"
...
 ```
 
-The `require` function can be used to require another namespace into the repl. The arguments are the same as the `:require` statement in the `ns` function.
+The `require` function can be used to require another namespace into the REPL. The arguments are the same as the `:require` statement in the `ns` function.
 ```bash
->>> (require phel\html :as h)
+phel:1> (require phel\html :as h)
 phel\html
->>> (h/html [:span])
+phel:3> (h/html [:span])
```
Comment (Chemaclass) [inline doc/content/documentation/repl.md:54]: `phel:2`

File: doc/content/documentation/repl.md (hunk 3 above)
Comment (jenshaase) [reply doc/content/documentation/repl.md:26]: Uh... There is a difference between `Ctrl+D` and `Ctrl+C` on how the program is terminated. Maybe it is better your google it. I probably explain it wrong.

Comment (JesusValera) [reply doc/content/documentation/repl.md:26]: https://dev.to/konkit/obvious-and-not-so-obvious-bash-zsh-terminal-shortcuts-3480#:~:text=Ctrl%2DC%20and%20Ctrl%2DD,D%20exits%20the%20REPL%20itself. :)
